### ISSUE TITLE
assign to template - display method is not working

### ISSUE THREAD

**Issue by liomj** [XoopsModules25x/mymenus#21]
<html><body>
<p dir="auto">referring to this issue <a class="issue-link js-issue-link" data-error-text="Failed to load title" data-id="842914583" data-permission-text="Title is private" data-url="https://github.com/XOOPS/XoopsCore25/issues/1011" data-hovercard-type="issue" data-hovercard-url="/XOOPS/XoopsCore25/issues/1011/hovercard" href="https://github.com/XOOPS/XoopsCore25/issues/1011">XOOPS/XoopsCore25#1011</a></p>
<p dir="auto">i cant get the assign to template working in new mymenus module, the menu is not displaying using the assign to template method<br>
My menus 1.54.0 Beta 2</p>

XOOPS Version | XOOPS 2.5.11-Beta2
-- | --
PHP Version | 7.4.24
MySQL Version | 5.5.5-10.4.21-MariaDB

but if i used the old mymenu module 1.3 with XOOPS 2.5.11-Beta2 the assign to template method works

**Comment by mambax7** [comment]
@liomj Is this still an issue? Can we close it?

**Comment by liomj** [comment]
yes solved in Beta 7 https://xoops.org/modules/newbb/viewtopic.php?post_id=367321#367321....
